lib/ReactRotatingText.js

```diff
@@ -200,9 +218,10 @@ ReactRotatingText.defaultProps = {
   deletingInterval: 50,
   emptyPause: 1000,
   eraseMode: 'erase',
-  items: ['first', 'second', 'third'],
+  items: ['first', 'second', 'third', 'fourth', 'fifth'],
```

Comment:
Were these values intentionally updated? 
Not that it would cause any problems but maybe should be a separate change if this is what we want :+1:

Reply:
@charj I believe these are updates from previously merged PRs however the lib was never re-built.
See https://github.com/adrianmcli/react-rotating-text/pull/14/files